<ui version="4.0" >
 <class>SearchDialog</class>
 <widget class="QMainWindow" name="SearchDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Strigi Search Dialog</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="1" >
     <widget class="QStackedWidget" name="mainview" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="page" >
       <layout class="QGridLayout" >
        <item row="0" column="0" colspan="2" >
         <widget class="QLabel" name="statusview" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="margin" >
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QPushButton" name="toggleindexing" >
          <property name="text" >
           <string>Stop Indexing</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="toggledaemon" >
          <property name="text" >
           <string>Stop Daemon</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2" >
         <widget class="QListWidget" name="indexeddirs" />
        </item>
        <item row="3" column="0" >
         <widget class="QPushButton" name="adddir" >
          <property name="text" >
           <string>Add Directory</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QPushButton" name="removedir" >
          <property name="text" >
           <string>Remove Directory</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QFrame" name="frame" >
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="SearchTabs" native="1" name="tabs" />
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="queryfield" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>About</string>
    </property>
    <addaction name="actionAbout_Strigi" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionEdit_filters" />
    <addaction name="actionList_indexed_files" />
   </widget>
   <widget class="QMenu" name="menuDisplay" >
    <property name="title" >
     <string>Display</string>
    </property>
    <addaction name="actionDisplay_Histogram" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuDisplay" />
   <addaction name="menuAbout" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionExit" >
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout_Strigi" >
   <property name="text" >
    <string>About Strigi</string>
   </property>
  </action>
  <action name="actionEdit_filters" >
   <property name="text" >
    <string>Edit filters</string>
   </property>
  </action>
  <action name="actionList_indexed_files" >
   <property name="text" >
    <string>List indexed files</string>
   </property>
  </action>
  <action name="actionDisplay_Histogram" >
   <property name="text" >
    <string>Hide &amp;Histogram</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchTabs</class>
   <extends>QWidget</extends>
   <header>searchtabs.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
